src/java.desktop/share/classes/javax/swing/text/MaskFormatter.java

```diff
@@ -25,10 +25,13 @@
 
 package javax.swing.text;
 
-import java.io.*;
-import java.text.*;
-import java.util.*;
-import javax.swing.*;
+import javax.swing.JFormattedTextField;
+import javax.swing.text.DefaultFormatter;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.Serial;
+import java.text.ParseException;
+import java.util.ArrayList;
```

Comment:
Could you please sort the imports?
In JDK code, `javax.*` packages follow `java.*` ones. So, the imports from `javax.swing` would be in the end of the list.